### PR TITLE
feat(runtime): add rspack version

### PR DESCRIPTION
**Pull request by LingyuCoder**
## Summary

Add rspack version:
- Add `__rspack_version__` api to get current rspack version in runtime
- Manually specific version value by `experiments.rspackFuture.bundlerInfo.version`
- Use `experiments.rspackFuture.bundlerInfo.force=true|string[]` to inject runtime module even when user do not use it, this feature is useful for debugging

## Test Plan

- Add `packages/rspack/tests/configCases/version`

## Require Documentation?

- [ ] No
- [x] Yes, the corresponding rspack-website PR is \_\_

### CODE REVIEW

File: crates/rspack_core/src/runtime_globals.rs [hunk 1]
```diff
@@ -245,6 +245,9 @@ bitflags! {
     const PRELOAD_CHUNK = 1 << 60;
 
     const PRELOAD_CHUNK_HANDLERS = 1 << 61;
+
+    // rspack only
+    const RSPACK_VERSION = 1 << 62;
   }
```
Comment (hardfist) [inline crates/rspack_core/src/runtime_globals.rs:251]: will it conflict with future webpack new runtime_globals?

Comment (LingyuCoder) [reply crates/rspack_core/src/runtime_globals.rs:251]: I think not, if webpack use `__webpack_require__.rv`, we should just change to another. It only affects the result of code generation, users will not feel the change.

**Comment by hardfist** [comment]
!canary

**Comment by KyrieLii** [comment]
LGTM 🫡

**Review by hardfist — APPROVED** [review]

**Pull request merged**